miden-lib/asm/note_scripts/P2ID.masm

```diff
@@ -52,28 +63,25 @@ proc.add_note_assets_to_account
     drop dropw drop
 end
 
-# Pay-to-ID script: adds all assets from the note to the account, assuming ID of the account
-# matches target account ID specified by the note inputs.
-#
-# Requires that the account exposes: miden::contracts::wallets::basic::receive_asset procedure.
-#
-# Inputs: [SCRIPT_ROOT]
-# Outputs: []
-#
-# Note inputs are assumed to be as follows:
-# - target_account_id is the ID of the account for which the note is intended.
-#
-# FAILS if:
-# - Account does not expose miden::contracts::wallets::basic::receive_asset procedure.
-# - Account ID of executing account is not equal to the Account ID specified via note inputs.
-# - The same non-fungible asset already exists in the account.
-# - Adding a fungible asset would result in amount overflow, i.e., the total amount would be
-#   greater than 2^63.
+#! Pay-to-ID script: adds all assets from the note to the account, assuming ID of the account
+#! matches target account ID specified by the note inputs.
+#!
+#! Requires that the account exposes: 
+#! - miden::contracts::wallets::basic::receive_asset procedure.
+#!
+#! Inputs:  [SCRIPT_ROOT]
+#! Outputs: []
```

Comment:
The inputs part needs to be updated as we no longer expect script root on the stack.